exchanges/gateio/gateio_websocket.go

```diff
@@ -52,12 +52,15 @@ const (
 
 	subscribeEvent   = "subscribe"
 	unsubscribeEvent = "unsubscribe"
+
+	// subscriptionBatchCount is the number of subscriptions to send in a single batch
+	subscriptionBatchCount = 200
 )
 
 var defaultSubscriptions = subscription.List{
 	{Enabled: true, Channel: subscription.TickerChannel, Asset: asset.Spot},
 	{Enabled: true, Channel: subscription.CandlesChannel, Asset: asset.Spot, Interval: kline.FiveMin},
-	{Enabled: true, Channel: subscription.OrderbookChannel, Asset: asset.Spot, Interval: kline.HundredMilliseconds},
+	{Enabled: true, Channel: subscription.OrderbookChannel, Asset: asset.Spot, Interval: kline.HundredMilliseconds, Levels: 100},
```

Comment:
This change like what I talked about in your other PR, has issues using `master` based subscriptions
```
[ERROR] | WEBSOCKET | 29/04/2025 10:46:15 | GateIO Error subscribing orderbook spot [ALEPH_USDT]: (2) provided level not supported: 0, orderbook spot [OGN_USDT]: (2) provided level not supported: 0, orderbook spot [ALA_USDT]: (2) provided level not supported: 0, orderbook spot [BTC_USDT]: (2) provided level not supported: 0
```
I don't mind where it gets fixed, but this PR is blocked until its reverted/fixed here/fixed elsewhere

Reply:
Yeah I will revert this 👍